README.md

```diff
@@ -34,6 +34,10 @@ Our goals include:
 - Accessibility features for users with disabilities
 - Regular updates and maintenance
 
+## Documentation
+
+For an easy-to-read overview of the project, visit the [BetterGov Docs](https://better-gov-docs.vercel.app/docs/about).
```

Comment:
Let's set up docs.bettergov.ph to maintain consistency. Let me know once you've added it to vercel